zabbix.yaml

```diff
@@ -1,6 +1,11 @@
 ---
-:zabbix_host: zabbix.domain.tld
-:zabbix_port: 10051
+:zabbix_host:
```

Comment:
This should be `:zabbix_hosts`, keep old `:zabbix_host` unchanged, and just merge it to the zabbix_host array inside the login